source/github-helpers/timeline-item.tsx

```diff
@@ -1,6 +1,11 @@
 import React from 'dom-chef';
 
-export default function TimelineItem(): JSX.Element {
+export function TimelineItemOld(): JSX.Element {
 	// Classes copied from #issuecomment-new + mt-3 added
 	return <div className="ml-0 pl-0 ml-md-6 pl-md-3 mt-3" />;
 }
+
+// #8141
+export function TimelineItem(): JSX.Element {
+	return <div className="ml-0 pl-0 pl-md-3 mt-3" />;
```

Comment:
Let's make sure it works correctly on all window widths. These classes were for the old version